concepts/observability-strategy/configurable-monitoring/04-monitoring-api.md

```diff
@@ -7,23 +7,32 @@ apiVersion: telemetry.kyma-project.io/v1alpha1
 metadata:
   name: PrometheusRemoteWrite
 spec:
-  receiver: # singular, different receivers need different pipelines and exporters configs
-    type: otlp # otlp  | custom, default is otlp
-    otlp: # maps to central otlp receiver
+  input: # Metrics can be received from different receivers (OTLP and Prometheus), metrics are selected based on labels
+    application:
       namespaces: # maps to filterprocessor
         include: []
         exclude: []
+        system: true
       containers:
         include: []
         exclude: []
-      system:
-        apiserver: 
-          enabled: true
-        nodes:
-          enabled: true
-        kube-state-metrics:
-          enabled: true
-          deployments: false
+      kubeStateMetrics:
```

Comment:
What is with all that other resource types.. maybe should be located under "system" as well?

Reply:
I changed the structure a bit to distinguish between `system` and `infrastructure`. The idea is that users can find anything that is related to their workload under `application`. Users should for instance be able to build a "golden signals" dashboard for their application using this metrics. Metrics under `system` contain everything that is provided by us, but related to a user workload. Everything under `infrastructure` is about the health of the Kubernetes cluster and nodes, which is not related to any user workload.